api/rest/v1/observations_point.md

```diff
@@ -0,0 +1,126 @@
+   For querying multiple variables or entities, see the [bulk version](/api/rest/v1/bulk/observations/point) of this endpoint.
+</div>
+
+## Request
+
+GET https://api.datacommons.org/v1/observations/point/{ENTITY_DCID}/{VARIABLE_DCID}
+{: #api-signature}
+
+<script src="/assets/js/syntax_highlighting.js"></script>
+
+### Parameters
+
+#### Path Parameters
+
+| Name                                                       | Description                                     |
+| ---------------------------------------------------------- | ----------------------------------------------- |
+| VARIABLE_DCID <br /> <required-tag>Required</required-tag> | DCID of the variable to query a value for.      |
+| ENTITY_DCID <br /> <required-tag>Required</required-tag>   | DCID of the entity that the variable describes. |
+{: .doc-table }
+
+#### Query Parameters
+
+| Name                                              | Type | Description                                                                                                                                                                                                                                                             |
+| ------------------------------------------------- | ---- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| date <br /> <optional-tag>Optional</optional-tag> | type | Datetime of measurement of the value requested in ISO 8601 format. To see the dates available, look up the variable in the [Statistical Variable Explorer](https://datacommons.org/tools/statvar). If date is not provided, the latest available datapoint is returned. |
+{: .doc-table }
+
+## Response
+
+The response will look something like:
+
+```json
+{
+ "date": "YYYY-MM-DD",
+ "value": 1234,
+ "facet": {...},
+}
+```
+
+### Response fields
+
+| Name  | Type   | Description                                                                                                                                        |
+| ----- | ------ | -------------------------------------------------------------------------------------------------------------------------------------------------- |
+| value | type   | Value of the variable queried for the queried entity.                                                                                              |
+| date  | string | Datetime the value returned was measured.                                                                                                          |
+| facet | dict   | Metadata on the [facet](/api/rest/v1/getting_started#facet) the data came from. Can include things like provenance, measurement method, and units. |
+{: .doc-table}
+
+## Examples
+
+### Example 1: Get single value for given variable and entity
```

Comment:
"Get latest observation for given variable and entity" to echo below "specific date"